.venv/bin/getST.py

```
@@ -0,0 +1,743 @@
+        self.__saveFileName = None
+        if options.hashes is not None:
+            self.__lmhash, self.__nthash = options.hashes.split(':')
+
+    def saveTicket(self, ticket, sessionKey):
+        logging.info('Saving ticket in %s' % (self.__saveFileName + '.ccache'))
+        ccache = CCache()
+
+        ccache.fromTGS(ticket, sessionKey, sessionKey)
+        ccache.saveFile(self.__saveFileName + '.ccache')
+
+    def doS4U2ProxyWithAdditionalTicket(self, tgt, cipher, oldSessionKey, sessionKey, nthash, aesKey, kdcHost, additional_ticket_path):
```

Ruff (0.14.2)

Unused method argument: `oldSessionKey`

(ARG002)